scripts/fast-reboot

```diff
@@ -663,6 +663,17 @@ if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
     fi
 fi
 
+TEAMD_INCREASE_RETRY_COUNT=0
+if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; then
+    /usr/local/bin/teamd_increase_retry_count.py --probe-only || TEAMD_RETRY_COUNT_PROBE_RC=$?
+    if [[ TEAMD_RETRY_COUNT_PROBE_RC -ne 0 ]]; then
+        debug "Warning: Not all neighbors support teamd retry count feature"
```

Comment:
This warning is not always true when return code is != 0. For example, the script can fail for reasons other than retry count support being unavailable.

Reply:
Updated the message to say that support is unknown for one or more neighbor devices.